common/data/src/main/java/org/thingsboard/server/common/data/widget/WidgetTypeInfo.java

```diff
@@ -65,5 +87,20 @@ public WidgetTypeInfo(WidgetTypeDetails widgetTypeDetails) {
         } else {
             this.widgetType = "";
         }
+        this.bundles = Collections.emptyList();
     }
+
+    public WidgetTypeInfo(WidgetTypeDetails widgetTypeDetails, List<EntityInfo> bundles) {
```

Comment:
Why copy-paste? Reuse in `WidgetTypeInfo(WidgetTypeDetails widgetTypeDetails)` (`this(widgetTypeDetails, Collections.emptyList();)`)